x-pack/plugins/apm/server/routes/alerts/rule_types/get_service_group_fields.test.ts

```diff
@@ -106,12 +108,14 @@ describe('getSourceFieldsAgg', () => {
 });
 
 describe('flattenSourceDoc', () => {
-  it('should flatten a given nested object with dot delim paths as keys', () => {
+  it('should flatten a given nested object with dot delim paths as keys except for labels', () => {
     const result = flattenSourceDoc(mockSourceObj);
     expect(result).toMatchInlineSnapshot(`
       Object {
         "agent.name": "nodejs",
-        "labels.team": "test",
+        "labels": Object {
+          "team": "test",
+        },
```

Comment:
It makes sense to change the `mockSourceObj` by adding a new label with an array as a value. 

Currently, you're not testing the `if-clause` you added on https://github.com/elastic/kibana/pull/177412/files#diff-8fbb5fff00430e6f04b6738321ce8b8cdd4cdacbd8aa9d69f855bd3e4b033ab4R55

Reply:
Good point, updated the tests.